tests/unit_tests/udf_wrapper_params/sequence_classification/MockSequenceClassification.py

```diff
@@ -0,0 +1,36 @@
+import torch
+from typing import Dict
+from dataclasses import dataclass
+
+
+class MockSequenceClassification:
+    @dataclass
+    class Config:
+        id2label: Dict[int, str]
+    config = Config(id2label={
+        0: 'label_1', 1: 'label_2',
+        2: 'label_3', 3: 'label_4'})
+
+    cnt = 0
+    model_based_data_size_at_each_batch = []
+
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+        # initiated by tests
+        if 'model_based_data_size_at_each_batch' in kwargs:
+            MockSequenceClassification.cnt = 0
+            MockSequenceClassification.model_based_data_size_at_each_batch = \
+                kwargs['model_based_data_size_at_each_batch']
+
+    @classmethod
```

Comment:
I think, we don't need a class method here, we can give the SequenceClassificationSingleText also a factory class instead. And, then we probably don't need the cnt class property, anymore.